src/common/ui-wrapper.js

```diff
@@ -50,8 +49,8 @@ class UIWrapper {
     this.setConfig(Utils.Object.mergeDeep({}, previewThumbnailConfig, seekbarConfig), 'seekbar');
   }
 
-  _handleVr(config: ?PKPluginsConfigObject): void {
-    if (config) {
+  _handleVr(config: PKPluginsConfigObject = {}): void {
+    if (config.vr && !config.vr.disable) {
```

Comment:
Actually, I would like to be sure the plugin is active (maybe its bundle non exist at all?) 
But this info is unavailable in kalturaPlayer yet.